safe_transaction_service/contracts/management/commands/update_safe_contracts_logos.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management import BaseCommand
+
+import requests
+
+from gnosis.eth import EthereumClientProvider
+
+from safe_transaction_service.contracts.models import Contract
+
+
+class Command(BaseCommand):
+    help = "Update safe contract logos by new one"
+    safe_deployments_base_url = (
+        "https://raw.githubusercontent.com/safe-global/safe-deployments/main/src/assets"
+    )
+    deployment_paths = {
```

Comment:
I think to make this work propertly we should have the information in safe-eth-py. Having this adhoc here is duplicating efforts and will be confusing in the future. Let's work on that here: https://github.com/safe-global/safe-eth-py/issues/622